pkg/cache/task.go

```diff
@@ -415,8 +408,11 @@ func (task *Task) sanityCheckBeforeScheduling() error {
 
 func (task *Task) enterState(event *fsm.Event) {
 	log.Logger.Info("task state",
+		zap.String("currentState", event.Src),
+		zap.String("targetState", event.Dst),
+		zap.String("event", event.Event),
 		zap.String("appID", task.applicationID),
 		zap.String("taskID", task.taskID),
 		zap.String("taskAlias", task.alias),
-		zap.String("state", task.GetTaskState()))
+		zap.String("currentTaskState", task.GetTaskState()))
```

Comment:
This is a double up: we have current and target state already (currentAppState).
Log at a debug level and with same info order: taskID, source, destination, and event. If we really need the appID to track it as the task ID is not unique then we should add that back.

Reply:
We need appID, taskID and alias for tracking purpose, revised to

```
	log.Logger.Debug("shim task state transition",
		zap.String("app", task.applicationID),
		zap.String("task", task.taskID),
		zap.String("taskAlias", task.alias),
		zap.String("source", event.Src),
		zap.String("destination", event.Dst),
		zap.String("event", event.Event))
```